osu.Game/Screens/Edit/EditorLoader.cs

```diff
@@ -65,6 +65,8 @@ protected override void LoadComplete()
             base.LoadComplete();
 
             // will be restored via lease, see `DisallowExternalBeatmapRulesetChanges`.
+            if (!(Beatmap.Value is DummyWorkingBeatmap))
```

Comment:
`Beatmap.Value` can't possibly be a `DummyWorkingBeatmap` in editor, can it? `Editor` creates a new working beatmap if it sees it:

https://github.com/ppy/osu/blob/f8830c6850128456266c82de83273204f8b74ac0/osu.Game/Screens/Edit/Editor.cs#L154-L170

Reply:
It can and not having this change fails tests. This is `EditorLoader`, not `Editor`.

Reply:
Aha, so it's done for tests and not real in-game scenarios.